source/apiVolontaria/volunteer/tests/tests_view_Events.py

```diff
@@ -242,6 +242,47 @@ def test_list_events_with_permissions(self):
             content['results'][2]['start_date']
         )
 
+    def test_list_events_with_filter(self):
+        """
+        Ensure we can list event filtered by cycle.
+        """
+        self.client.force_authenticate(user=self.admin)
+
+        url = "{0}?cycle={1}".format(
```

Comment:
Je ne sais pas mais "existe il plus propre que de hardcode le param comme ça? "

Reply:
Qu'est-ce que tu suggérerais?

Reply:
Yeah, i searched and found nothing to add params as an attribute of reverse... So it's good for me :+1: